types/react-native/index.d.ts

```diff
@@ -8779,7 +8779,33 @@ export namespace Animated {
         // Internal class, no public API.
     }
 
-    class AnimatedWithChildren extends Animated {
```

Comment:
To confirm, extending `Animated` was unimportant or incorrect?

Reply:
@amcasey 
yes, AnimatedWithChildren extends AnimatedNode in the original js file
it shouldn't negatively affect the users, as the extended Animated type did not include any public methods (in terms of typing)

https://github.com/facebook/react-native/blob/master/Libraries/Animated/nodes/AnimatedWithChildren.js